### DISCUSSION TITLE
Archive or Delete folder

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#2589] [Q&A]
When clicking FOLDERS in the menu, if I highlight a folder there is no option to archive or delete it. I can go into the folder and archive individual photos but not the entire folder.  Is there a way to accomplish this?

**Comment by ghost** [comment]
and I know the documentation says this, but it is not available:
The context menu allows you to perform the following actions:
Remove Folders[¶](https://docs.photoprism.app/user-guide/organize/folders/#remove-folders)

    Select folder
    Open context menu
    Click

Confirm

**Comment by lastzero** [comment]
It's a folder album, including all the pictures in that folder on your disk. You can delete it by deleting that folder on your disk.
